DigitalOffice/Services/TimeManagementService/Validators/CreateWorkTimeRequestValidator.cs

```diff
@@ -0,0 +1,89 @@
+﻿using FluentValidation;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using LT.DigitalOffice.TimeManagementService.Models;
+using LT.DigitalOffice.TimeManagementService.Repositories.Filters;
+using LT.DigitalOffice.TimeManagementService.Repositories.Interfaces;
+using System.Globalization;
+
+namespace LT.DigitalOffice.TimeManagementService.Validators
+{
+    public class CreateWorkTimeRequestValidator : AbstractValidator<CreateWorkTimeRequest>
+    {
+        private IWorkTimeRepository repository;
+
+        /// <summary>
+        /// How many days ago can WorkTime be added.
+        /// </summary>
+        public const int fromDay = 3;
+        /// <summary>
+        /// How many days ahead can WorkTime be added.
+        /// </summary>
+        public const int toDay = 2;
```

Comment:
The recommended naming and capitalization convention is to use PascalCasing for constants.

Reply:
actually, UPPER_SNAKE_CASE;
I think @KostinP  was one of those to suggest that :D

Reply:
UPPER_SNAKE_CASE looks bad

Reply:
When we talked about this, you said that Microsoft uses the proposed top option, can you write examples?